static/js/drafts.js

```diff
@@ -183,6 +184,16 @@ exports.setup_page = function (callback) {
         });
     }
 
+    function remove_old_draft() {
+        var old_date  = new Date().setDate(new Date().getDate() - draft_lifetime);
```

Comment:
This function should have some sort of unit tests.  A particular issue I'd want the tests to ensure we don't have is cases where the subtraction is in some unit other than days (e.g. seconds!), since that's a classic bug for a feature like this.